config/config.go

```diff
@@ -1391,7 +1410,8 @@ type RemoteWriteConfig struct {
 
 	// We cannot do proper Go type embedding below as the parser will then parse
 	// values arbitrarily into the overflow maps of further-down types.
-	HTTPClientConfig HTTPClientConfig `yaml:",inline"`
+	HTTPClientConfig   HTTPClientConfig   `yaml:",inline"`
+	QueueManagerConfig QueueManagerConfig `yaml:",inline"`
```

Comment:
I think this is a section which will get many more options in the future, and which may be extra-experimental so my instinct would be to not inline it.